markdownlint.js

```diff
@@ -36,15 +36,19 @@ function yamlParse(text) {
   return require('js-yaml').load(text);
 }
 
+function tomlParse(text) {
+  return require('toml').parse(text);
+}
+
 const exitCodes = {
   lintFindings: 1,
   failedToWriteOutputFile: 2,
   failedToLoadCustomRules: 3,
   unexpectedError: 4
 };
 
-const projectConfigFiles = ['.markdownlint.jsonc', '.markdownlint.json', '.markdownlint.yaml', '.markdownlint.yml'];
-const configParsers = [jsoncParse, yamlParse];
+const projectConfigFiles = ['.markdownlint.jsonc', '.markdownlint.json', '.markdownlint.yaml', '.markdownlint.yml', '.markdownlint.toml'];
```

Comment:
I'd prefer not to introduce TOML as an officially supported/searched-for configuration file name because it is NOT supported by CLI2 or the VS Code extension.

Reply:
Fair point. I removed it.